tsfc/fem.py

```diff
@@ -522,11 +543,12 @@ def callback(key):
 
 
 def coordinate_coefficient(domain):
+    """Create a fake coordinate coefficient for a domain."""
     return ufl.Coefficient(ufl.FunctionSpace(domain, domain.ufl_coordinate_element()))
 
 
 def replace_coordinates(integrand, coordinate_coefficient):
-    # Replace SpatialCoordinate nodes with Coefficients
+    """Replace SpatialCoordinate nodes with Coefficients."""
     return map_expr_dag(ReplaceSpatialCoordinates(coordinate_coefficient), integrand)
```

Comment:
Do we expect a significant amount of shared* expressions between integrands? I believe otherwise it shouldn't really matter.

*_Shared_ as in _object identity_, not as in _equal_.